locals.tf

```diff
@@ -1,140 +0,0 @@
-terraform {
-  experiments = [module_variable_optional_attrs]
-}
-locals {
```

Comment:
we don't need to wrap all of these in locals, only those that are re-used to avoid duplication